nifi-api/src/main/java/org/apache/nifi/reporting/Bulletin.java

```diff
@@ -131,6 +132,14 @@ public void setSourceType(ComponentType sourceType) {
         this.sourceType = sourceType;
     }
 
+    public String getFlowFileUUID() {
+        return flowFileUUID;
+    }
+
+    public void setFlowFileUUID(String flowFileUUID) {
+        this.flowFileUUID = flowFileUUID;
+    }
```

Comment:
Java naming conventions are generally accepted to use mixed case with acronyms (i.e., getFlowFileUuid(), setFlowFileUuid()). This is definitely the pattern we tend to follow in NiFi. Probably best to use that naming convention here.